SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1456 @@
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$user,
+        [Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [String]$pass
+    )
+	
+	$passwordPolicyNodes = New-Object System.Collections.Generic.List[System.Object]
+
+	Try {
+		if (Test-WSAConnection -server $server) {
+			$WSAToken = Request-WSAToken -fqdn $server -user $user -pass $pass
+			if ($WSAToken) {
+				$wsaPasswordPolicy = Get-WSAPasswordPolicy | Select-Object * -ExcludeProperty _links
+				$wsaPasswordLockout = Get-WSAPasswordLockout | Select-Object * -ExcludeProperty _links
+		
+				$wsaTmpObject = @()
+				$wsaTmpObject += [pscustomobject]@{
+				"passwdExpInDays" = [int]$wsaPasswordPolicy.passwordTtlInHours / 24
+				"passwdHistoryRestriction" = $wsaPasswordPolicy.history
+				"passwdMinimumLength" = $wsaPasswordPolicy.minLen
+				"passwdMinUppercase" = $wsaPasswordPolicy.minUpper
+				"passwdMinLowercase" = $wsaPasswordPolicy.minLower
+				"passwdMinNumeric" = $wsaPasswordPolicy.minDigit
+				"passwdMinSpecial" = $wsaPasswordPolicy.minSpecial
+				"passwdMaxConsecutiveIdenticalChar" = $wsaPasswordPolicy.maxConsecutiveIdenticalCharacters
+				"passwdMaxPreviousCharactersReused" = $wsaPasswordPolicy.maxPreviousPasswordCharactersReused
+				"passwdMaxFailAttempts" = $wsaPasswordLockout.numAttempts
+				"passwdAttemptsIntervalInMins" = $wsaPasswordLockout.attemptInterval
+				"passwdUnlockIntervalInMins" = $wsaPasswordLockout.unlockInterval
+				"tempPasswdLifetimeInHour" = $wsaPasswordPolicy.tempPasswordTtl
+				"passwdExpReminderInDay" = [int64]$wsaPasswordPolicy.notificationThreshold / 86400000
+				"passwdExpReminderNotificationFrequencyInDay" = [int64]$wsaPasswordPolicy.notificationInterval / 86400000
+				}
+				
+				$nodeConfigValue = New-Object -TypeName psobject
+				$nodeConfigValue | Add-Member -notepropertyname "FQDN" -notepropertyvalue $server
+				$nodeConfigValue | Add-Member -notepropertyname "productType" -notepropertyvalue "WSA"
+				$nodeConfigValue | Add-Member -notepropertyname "productAttributes" -notepropertyvalue ($wsaTmpObject | Select-Object -Skip 0)
+				$passwordPolicyNodes.Add($nodeConfigValue)
+				return $passwordPolicyNodes
+			} else {
+				Write-LogMessage -Type Error -Message "Failed to obtain access token from Workspace ONE Access server '$server',  please verify your credentials and try again"
+			}
+		} else {
+				Write-LogMessage -Type Error -Message "Testing a connection to Workspace ONE Access server $server failed, please check your details and try again"
+		}
+    }
+    Catch {
+        Debug-ExceptionWriter -object $_
+    }
+}
+
+Function GetEnvironmentPasswordPolicyDetail {
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$ppmVariables,
+        [Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmEnvironmentalDetails
+	)
+	if ($ppmVariables.isEnvDetailSet -eq $false) {
```

Comment:
Add line space after Param block

Reply:
addressed in 9ee41934dd0ee5a302dd128339b72e2f82898175